tests/integration.spec.coffee

```diff
@@ -1722,6 +1722,16 @@ describe 'SDK Integration Tests', ->
 						promise = resin.models.os.getDownloadSize('raspberrypi')
 						m.chai.expect(promise).to.eventually.satisfy((n) -> typeof n is 'number')
 
+				describe 'given a specific OS version', ->
+
+					it 'should get a result for ResinOS v1', ->
+						promise = resin.models.os.getDownloadSize('raspberry-pi', '1.24.0')
```

Comment:
BTW any chance to test that memoization works?
One suggestion is to call the `getDownloadSize` for two different versions and ensure the results are different. That as the very least will ensure that the version does actually influence the cache key.